etc/neutron/services/f5/f5-openstack-agent.ini

```diff
@@ -292,6 +292,11 @@ l2_population = True
 #
 # f5_network_segment_gross_timeout = 300
```

Comment:
The documentation needs to change when we have more clarity on how opflex networks are to be scanned for segments.  I changed the polling timeout to 60 seconds.